assignment/tool/src/java/org/sakaiproject/assignment/entityproviders/AssignmentEntityProvider.java

```diff
@@ -555,8 +555,11 @@ public ActionReturn getGrades(Map<String, Object> params) {
             Set<AssignmentSubmissionSubmitter> submitters = s.getSubmitters();
 
             if (submitters.size() > 0) {
-                //String rawGrade = assignmentService.getGradeForSubmitter(s, submitters.iterator().next().getSubmitter());
-                return assignmentService.getGradeDisplay(s.getGrade(), assignment.getTypeOfGrade(), assignment.getScaleFactor());
+                if (assignment.getTypeOfGrade() == Assignment.GradeType.PASS_FAIL_GRADE_TYPE) {
+                    return s.getGrade() == null ? "ungraded" : s.getGrade();
```

Comment:
Is the "graded" string used in the UI?

Reply:
No, ungraded is literally just for the new grader, the entity provider. It doesn't impact the existing UI in anyway. I didn't want to do matches on translated strings, as that is just a terrible pattern and needs to be stripped out of assignments.